baselines/diabetic_retinopathy_detection/dropout_tune_on_retention_auc.py

```diff
@@ -145,7 +158,9 @@ def main(argv):
   test_splits = [split for split in available_splits if 'test' in split]
   eval_splits = [split for split in available_splits
                  if 'validation' in split or 'test' in split]
-  eval_datasets = {split: datasets[split] for split in eval_splits}
+
+  # Iterate eval datasets
+  eval_datasets = {split: iter(datasets[split]) for split in eval_splits}
```

Comment:
We will be repeating eval datasets, so we can just run iter() once here, and not later